clusterloader2/pkg/measurement/common/probes/probes.go

```diff
@@ -124,14 +128,15 @@ func (p *probesMeasurement) String() string {
 	return probesMeasurementName
 }
 
-func (p *probesMeasurement) initialize(config *measurement.MeasurementConfig) error {
-	replicasPerProbe, err := util.GetInt(config.Params, "replicasPerProbe")
+func (p *probesMeasurement) initialize(c *measurement.MeasurementConfig) error {
```

Comment:
I actually prefer config than c - it's more meaningful.

Reply:
Sure, that's leftover from a previous iteration.